src/client/src/components/DraggableSidebarItem/index.tsx

```diff
@@ -114,6 +117,7 @@ const DraggableSidebarItem = ({
           )}
           {azureFunctions && <AzureFunctionsIcon />}
           {cosmosDB && <CosmosDBIcon />}
+          {appService && <AppServiceIcon />}
```

Comment:
Could you create a task to refactor all these boolean props to some kind of enum value? :)

Reply:
Where would you want the enum to be created?

Reply:
Constants file would be safest bet in case you have to use it anywhere else.

Reply:
Sounds good, task created here:

https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/31909